src/shared/dom-utility.ts

```diff
@@ -0,0 +1,44 @@
+export class BBAuthDomUtility {
+
+  public static addCss(css: string): HTMLStyleElement {
+    const styleEl = document.createElement('style');
+
+    styleEl.appendChild(document.createTextNode(css));
+
+    document.head.appendChild(styleEl);
+
+    return styleEl;
+  }
+
+  public static addIframe(
+    src: string,
+    className: string,
+    title: string
+  ): HTMLIFrameElement {
+    const iframeEl = document.createElement('iframe');
+    iframeEl.className = className;
+    iframeEl.title = title;
+    iframeEl.src = src;
+
+    this.addElToBodyTop(iframeEl);
+
+    return iframeEl;
+  }
+
+  public static removeCss(styleEl: HTMLStyleElement) {
+    document.head.removeChild(styleEl);
```

Comment:
Any concern about removing a child that may not exist?

Reply:
I just pushed a commit that addresses this.  Thanks for the reminder.